controllers/error.controllers.js

```diff
@@ -0,0 +1,8 @@
+exports.handleMissingEndpoints = (req,res,next) => {
+  return res.status(404).send({msg:`${req.url} endpoint not found`})
+}
+
+exports.handleServerErrors = (err,req,res,next) => {
+  console.log(err,'<--- in server error controller')
+  return res.status(500).send({msg:'internal error'})
+}
```

Comment:
Ideally these would be in their own folder so as not to get lost in your controllers. But as long as you know where they are for debugging :)